lib/session.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2017 Super Adventure Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/nafundi/super-adventure/blob/master/NOTICE.
+
+This file is part of Super Adventure. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of Super Adventure,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+import moment from 'moment';
+
+export default class Session {
+  get token() { return this._token; }
+  // TODO: Use `expires` in some way.
+  get expires() { return this._expires; }
```

Comment:
are you actually using this for something? or planning to?

Reply:
I was thinking that before each request, Super Adventure could check whether the token is expired. If it is, it doesn't send the request, but instead puts up an alert telling the user that they need to log in again.

Reply:
okay, fair enough.

Reply:
but you will also have to react if the token has been invalidated for any other reason.

Reply:
Sounds good. I'm still thinking through the best way to implement this, but I'm thinking we should implement it before MJP. What do you think?